### PR TITLE
fix(TDP-3175): forms with bootstrap theme

### PR DESCRIPTION
**Pull request by jsomsanith**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-3175

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [ ] The new code does not introduce new technical issues (sonar / eslint)
- [ ] Functional tests have been performed

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

**(Optional) What is the current behavior?**
(Additional information to the Jira)

**(Optional) What is the new behavior?**
(Additional information to the Jira)
Update bootstrap theme
Fix radio and checkbox + additional forms elements

### CODE REVIEW

File: dataprep-webapp/src/app/components/feedback/_feedback.scss [hunk 1]
```diff
@@ -39,20 +39,20 @@
 
   .feedback-form-line {
     @include flex-row();
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/feedback/_feedback.scss]: You miss it or you use default value?

File: dataprep-webapp/src/app/components/feedback/_feedback.scss [hunk 2]
```diff
@@ -39,20 +39,20 @@
...
+      flex-grow: 0;
+      flex-shrink: 0;
 
       text-align: right;
       padding-right: 10px;
     }
 
     select {
-      height: 36px;
+      height: 32px;
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/feedback/_feedback.scss]: /giphy scientific method

File: dataprep-webapp/src/app/components/suggestions-stats/column-profile-options/_column-profile-options.scss [hunk 3]
```diff
@@ -17,14 +17,18 @@
...
+    align-items: center;
 
     .chart-control-options {
       border: 1px solid;
       border-radius: 3px;
       padding: 5px;
       text-transform: uppercase;
 
+      label {
+        font-family: $sourcesanspro;
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/suggestions-stats/column-profile-options/_column-profile-options.scss]: Useless if we remove Helvetica from app

File: dataprep-webapp/src/app/components/suggestions-stats/column-profile-options/column-profile-options.html [hunk 4]
```diff
@@ -33,8 +33,8 @@
                         </select>
                     </div>
 
-                    <div ng-if="$ctrl.selected.column">
-                        <span translate-once="AGGREGATION"></span>
+                    <div ng-if="$ctrl.selected.column" class="form-group">
+                        <label translate-once="AGGREGATION"></label>
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/suggestions-stats/column-profile-options/column-profile-options.html]: You don't need `.control-label` and `.form-control` BS classes?

File: dataprep-webapp/src/app/components/widgets/slidable/_slidable.scss [hunk 5]
```diff
@@ -12,134 +12,134 @@
...
-    overflow-y: auto;
-
-    > .content-container {
-      min-width: $width;
-    }
-  }
+	$width: $size - $actionSize;
+	@include slidable-common($size, $actionSize);
+	flex-direction: row;
+	@include transition(0.20s ease flex);
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/widgets/slidable/_slidable.scss]: Why did you keep this mixin? Autoprefixer should do the stuff too.

File: dataprep-webapp/src/app/components/widgets/slidable/_slidable.scss [hunk 6]
```diff
@@ -12,134 +12,134 @@
...
-    overflow: auto;
-
-    > .content-container {
-      min-height: $height;
-    }
-  }
+	$height: $size - $actionSize;
+	@include slidable-common($size, $actionSize);
+	flex-direction: column;
+	@include transition(0.20s ease transform);
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/widgets/slidable/_slidable.scss]: Let Autoprefixer do stuff

File: dataprep-webapp/src/app/components/feedback/_feedback.scss (hunk 1 above)
Comment (jsomsanith) [inline dataprep-webapp/src/app/components/feedback/_feedback.scss]: This is one of our mixins. It adds `align-items: stretch` too

File: dataprep-webapp/src/app/components/suggestions-stats/column-profile-options/column-profile-options.html (hunk 4 above)
Comment (jsomsanith) [inline dataprep-webapp/src/app/components/suggestions-stats/column-profile-options/column-profile-options.html]: The whole forms in TDP should be reworked :) 
Out of this PR scope

File: dataprep-webapp/src/app/components/feedback/_feedback.scss (hunk 2 above)
Comment (jsomsanith) [inline dataprep-webapp/src/app/components/feedback/_feedback.scss]: :P

**Pull request merged**